packages/react-native-dogfood/src/screens/Meeting/GuestMeetingScreen.tsx

```diff
@@ -75,15 +70,7 @@ export const GuestMeetingScreen = (props: Props) => {
     user: userToConnect,
   });
 
-  useEffect(() => {
-    if (!activeCall) {
-      return;
-    }
-    startForegroundService();
-    return () => {
-      stopForegroundService();
-    };
-  }, [activeCall]);
+  usePushRegisterEffect(client);
```

Comment:
Same here, can we move calling `usePushRegisterEffect` to our StreamVideo to make it future-proof?